src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -152,4 +168,22 @@ private static boolean isSwitchExpression(DetailAST ast) {
         return ast.getParent().getType() == TokenTypes.EXPR
                 || ast.getParent().getParent().getType() == TokenTypes.EXPR;
     }
+
+    /**
+     * Checks if the case contains null label.
+     *
+     * @param ast the switch statement we are checking
+     * @param tokenType the tokenType of null literal
+     * @return returnValue the ast of null label
+     */
+    private static @Nullable DetailAST verifyNullCaseLabel(DetailAST ast, int tokenType) {
```

Comment:
This can be a simple boolean method with inlined return of conditions, please do this. We have no need to pass in token type here, there is only one place this is used. We can call this method `hasNullCaseLabel`.